pages/partners/farmlands.vue

```diff
@@ -0,0 +1,126 @@
+          target="_blank"
+          class="no-underline"
+        >
+          <div class="flex items-center px-4 md:pl-40 pt-0.5 pb-3">
+            <icons-external-link class="text-interactive-tertiary w-4 h-4" />
+            <div class="text-content-inverse-primary px-2">
+              Website
+            </div>
+          </div>
+        </NuxtLink>
+        <div class="flex items-center text-content-inverse-primary px-4 md:pl-40 pt-8 pb-16">
+          <div class="pr-3">
+            <NuxtLink
+              to="https://www.facebook.com/farmlandsnz/"
+              target="_blank"
+            >
+              <icons-facebook class="text-interactive-tertiary w-5 h-5" />
+            </NuxtLink>
+          </div>
+          <div class="px-3">
+            <NuxtLink
+              to="https://twitter.com/farmlands?lang=en"
+              target="_blank"
+            >
+              <icons-twitter class="text-interactive-tertiary w-5 h-5" />
+            </NuxtLink>
+          </div>
+          <div class="px-3">
+            <NuxtLink
+              to="https://www.instagram.com/farmlandsnz/?hl=en"
+              target="_blank"
+            >
+              <icons-instagram class="text-interactive-tertiary w-5 h-5" />
+            </NuxtLink>
+          </div>
+          <div class="px-3">
+            <NuxtLink
+              to="https://www.linkedin.com/company/farmlands-trading-society-ltd/?originalSubdomain=nz"
+              target="_blank"
+            >
+              <icons-linkedin class="text-interactive-tertiary w-5 h-5" />
+            </NuxtLink>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="desktop-gutters">
+      <div class="px-6 md:px-40">
+        <div class="text-content-primary text-4xl leading-10 font-extrabold tracking-tight pt-16 pb-4">
+          Overview
+        </div>
+        <div class="text-content-secondary type-body-1 py-4">
+          In partnership with Centrapay, Farmlands now offer Card Partners two new digital ways to accept and reconcile Farmlands Card payments. There are no additional costs to Card Partners to utilise these new digital tools provided by Centrapay.
+          <br><br><br>
+          Whether your business needs a solution fully-integrated with your retail point-of-sale system, or just access to a secure web portal to take payments anytime, anywhere - there is now an option for you.
+        </div>
+
+        <img
+          src="~/assets/images/farmlands-flow.png"
+          class="w-full m-auto pt-4 pb-16"
+        >
+      </div>
+      <div class="px-6 md:px-40">
+        <div class="text-content-primary text-4xl leading-10 font-extrabold tracking-tight pt-16">
+          Solutions
+        </div>
+
+        <div class="flex flex-col md:flex-row pt-12 md:space-x-6 pb-16 space-y-6 md:space-y-0">
+          <div
+            class="rounded-2xl md:w-1/2 bg-cover bg-[url('~/assets/images/pos-api-background.png')]"
```

Comment:
@AndyClifford I can't see the background image locally - can you try to clear you cache and make sure it still displays?
![image](https://user-images.githubusercontent.com/46909693/212172196-7162fab7-1912-4ad6-b131-af341d5956e8.png)

Reply:
Interesting - I can see it on my screen after clearing cache. Shows up in Chrome and Safari. What browser are you using?

Reply:
This has been sorted by downloading the image at its original size and cropping some of the bottom and top